doc/changelog/07-vernac-commands-and-options/14093-fix-14092.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  The Ltac2 grammar can now be printed using the
+  Print Grammar ltac2 command
```

Comment:
It only adds the ability to print the `ltac2_expr` nonterminal.  It doesn't include many things such as the various `q_*` symbols.

While I don't object to the change, the concept of `Print Grammar` is quite broken.  It certainly doesn't help users that Ltac2 has slightly different syntax for many build-in tactics.

Reply:
I think `Print Grammar` should be generalized to accept any grammar entry.  Maybe we should open an issue for this?

Reply:
I wouldn't recommend that.  We've spent a lot of time cleaning up the grammar we present in the documentation--clearly the right thing to do for the documentation.  Print Grammar doesn't match this and it would not be simple to make it match.  The currently printable nonterminals are the ones that can be altered at runtime by notations.

Another issue with Ltac2 is that many of the built-in tactics are not defined in .v files so they are not available unless the user defines them.  This will be an ongoing maintenance problem.